packages/components/slider/stories/slider.stories.tsx

```diff
@@ -13,7 +13,7 @@ export default {
     },
     color: {
       control: {type: "select"},
-      options: ["default", "primary", "secondary", "success", "warning", "danger"],
+      options: ["foreground", "primary", "secondary", "success", "warning", "danger"],
```

Comment:
`foreground` might be a better name, but keeping it consistent is IMO more important.

Reply:
We use `foreground` in some components https://nextui.org/docs/components/link#colors

Reply:
Okay :+1: